src/index.js

```diff
@@ -91,10 +95,22 @@ async function runtest(page, name) {
   // await runtest(page, "UploadPayrollStatus");
 
   await page.goto(
-    "https://srv136.services.gc.ca/ROE-RE/ROEWeb-REWeb/pro/Requests/Prints?org_id=" + orgId
+    "https://srv136.services.gc.ca/ROE-RE/ROEWeb-REWeb/pro/Requests/Prints?org_id=" +
+      orgId
   );
   await runtest(page, "RequestedPrintFiles");
 
+  await page.goto(
+    "https://srv136.services.gc.ca/ROE-RE/ROEWeb-REWeb/pro/Search/Issued?org_id=" +
```

Comment:
master requires a non negative org id, rebasing should fix earlier issues, but you'll have to manually modify this one. 
to be org_id=-